R/utils-code_dependency.R

```diff
@@ -129,8 +134,17 @@ detect_symbol <- function(object, pd) {
     vapply(
       pd,
       function(call) {
-        any(call[call$token %in% c("SYMBOL", "SYMBOL_FUNCTION_CALL"), "text"] == object) &&
+        is_symbol <-
+          any(call[call$token %in% c("SYMBOL", "SYMBOL_FUNCTION_CALL"), "text"] == object) &&
           !any(call[call$token == "SYMBOL_FORMALS", "text"] == object)
+
+        object_ids <- call[call$text == object, 'id']
+        dollar_ids <- call[call$'token' == "'$'", 'id']
```

Comment:
let's include @ also
```suggestion
dollar_ids <- call[call$'token' %in% c("'$'", "'@'"), 'id']
```

Reply:
5f93d480d41bd2eaf83bdccbb0311a6e0f0cb0bc